app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args):
+        if len(cache_dict) < 1:
+            print("Calculating new result")
+            cache_dict[args] = func(*args)
+        else:
+            if args in cache_dict.keys():
+                print("Getting from cache")
+            else:
+                print("Calculating new result")
+                cache_dict[args] = func(*args)
+        return cache_dict.get(args)
```

Comment:
```suggestion
        return cache_dict[args]
```
More clear way to get value by key if you are sure it is present in the dictionary